src/index.html

```diff
@@ -14,40 +21,57 @@ <h1>2048</h1>
           <p class="info">
             Score: <span class="game-score">0</span>
           </p>
-          <button class="button start">Start</button>
+          <button id="startButton"  class="button start"
+          onclick="startGame()">Start</button>
```

Comment:
Fix code style everywhere
```suggestion
          <button
            id="startButton"
            class="button start"
            onclick="startGame()"
          >
            Start
          </button>
```